tests/integration/tier0/non-destructive/rhsm-facts/main.fmf

```diff
@@ -0,0 +1,21 @@
+summary+: |
+    RHSM facts update when in analysis mode
+description+: |
+    Verify that the subscription-manager facts command is called after the
+    analysis is done (either succeeding or failling).
+
+/rhsm_facts_update_in_analysis:
+    summary+: |
+        RHSM facts update after analysis
+    description+: |
+        TODO(r0x0d): ...
```

Comment:
Just don't forget to this TODO and also the empty doc string in the python code of this test